cmds/record.c

```diff
@@ -1554,6 +1554,9 @@ static void check_binary(struct opts *opts)
 		EM_X86_64, EM_ARM, EM_AARCH64, EM_386
 	};
 
+if ( !(opts->exename))
+	pr_err_ns("Cannot open '(null)' : Invalid filename.\n");
+
```

Comment:
Do you really need this?  It already checks the number of arguments and will print an error message if it misses exename.

Reply:
OK, then please write the exact command line to reproduce in the commit message.